### ISSUE TITLE
Title: FullyConnected + ReLU fusion broken in AI Edge Torch ≥ 0.4.0 (bias broadcasting prevents GPU execution)

### ISSUE THREAD

**Issue by kadu-v** [google-ai-edge/ai-edge-torch#717]
### Description of the bug:

nn.Linear (with bias) + nn.ReLU exports as separate FullyConnected and Bias+ReLU, dropping the batch dimension and breaking GPU execution in AI Edge Torch ≥ 0.4.0.

## Expected behaviour
- Converter fuses nn.Linear (with bias) and nn.ReLU into a single FullyConnected + ReLU op.
- Output tensor keeps its batch dimension: 1 × 64 × 5.

## Actual behaviour (≥ 0.4.0)
- Graph contains two ops:
    1.	FULLY_CONNECTED – output shape 64 × 5 (batch dim dropped)
    2. BIAS_ADD + RELU – bias constant shape 1 × 5
- Broadcast from 1 × 5 to 64 × 5 triggers an “unsupported broadcast” error on Mali GPUs, so the model falls back to CPU.

## Minimal example for reproduing the bug
```python
import ai_edge_torch
import tensorflow as tf
import torch
import torch.nn as nn


class Bug1Model(nn.Module):
    def __init__(self):
        super(Bug1Model, self).__init__()
        self.fc = nn.Linear(10, 5, bias=True)
        self.act = nn.ReLU()

    def forward(self, x):
        x = self.fc(x)
        x = self.act(x)
        return x


def main():
    model = Bug1Model()
    model = model.eval()

    sample_inputs = (torch.randn(1, 64, 10),)
    tfl_converter_flags = {"optimizations": [tf.lite.Optimize.DEFAULT]}
    edge_model = ai_edge_torch.convert(
        model, sample_args=sample_inputs, _ai_edge_converter_flags=tfl_converter_flags
    )
    edge_model.export("bug1.tflite")


if __name__ == "__main__":
    main()
```

> [!NOTE]
> A generated model and full example code are [here](https://github.com/kadu-v/ai-edge-torch-bugs-report)

**Comment by gaikwadrahul8** [comment]
Hi, @kadu-v 
Thank you for bringing this issue to our attention, I am able to replicate same behavior from my end so we will have to dig more into this issue and will update you soon, I'm keeping gist-file as reference for investigation purpose and also attached Netron graph below for reference. Thank you for your cooperation and understanding.

**Here is Netron graph output :**
![Image](https://github.com/user-attachments/assets/1d73443f-dd7d-4f0c-b1df-798ab810a34d)

**Comment by aravindmurali22** [comment]
Hi @suleshahid @turbotoribio , following up on this thread, is there any update on this issue? Thanks!
